src/modules/Profile.rb

```diff
@@ -13,26 +13,10 @@ class ProfileClass < Module
     # All these sections are handled by AutoYaST (or Installer) itself,
     # it doesn't use any external AutoYaST client for them
     GENERIC_PROFILE_SECTIONS = [
```

Comment:
I'm not getting this change in general. Are these previous assumptions incorrect? See below